src/libraries/System.Configuration.ConfigurationManager/tests/System/Configuration/SettingsPropertyTests.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Configuration;
+using Xunit;
+
+namespace System.ConfigurationTests
+{
+    public class SettingsPropertyTests
+    {
+        [Fact]
+        public void SettingsProperty_WithNoArguments()
+        {
+            var settingsProperty = new SettingsProperty("TestName");
+            Assert.Equal("TestName", settingsProperty.Name);
+            Assert.False(settingsProperty.IsReadOnly);
+            Assert.Null(settingsProperty.DefaultValue);
+            Assert.Null(settingsProperty.PropertyType);
+            Assert.Equal(SettingsSerializeAs.String, settingsProperty.SerializeAs);
+            Assert.Null(settingsProperty.Provider);
+            Assert.NotNull(settingsProperty.Attributes);
+            Assert.False(settingsProperty.ThrowOnErrorDeserializing);
+            Assert.False(settingsProperty.ThrowOnErrorSerializing);
+        }
+
+        [Theory]
+        [InlineData(SettingsSerializeAs.String)]
+        [InlineData(SettingsSerializeAs.Xml)]
+        [InlineData(SettingsSerializeAs.ProviderSpecific)]
+        public void SettingsProperty_WithArguments(SettingsSerializeAs serializeAs)
+        {
+            var settingsProperty = new SettingsProperty(
+                "TestName",
+                typeof(string),
+                null,
+                true,
+                "TestDefaultValue",
+                serializeAs,
+                new SettingsAttributeDictionary(),
+                true,
```

Comment:
Comment
nit: add parameter name - e.g. `throwOnErrorDeserializing : true` - here and elsewhere for the various bools\nulls.